src/command_modules/azure-cli-role/azure/cli/command_modules/role/tests/latest/test_role.py

```diff
@@ -285,7 +286,7 @@ def test_role_assignment_audits(self, resource_group):
                     self.cmd('role assignment list-changelogs --start-time {}'.format(start_time))
                 else:
                     # NOTE: get the time range from the recording file and use them below for playback
-                    start_time, end_time = '2018-02-10T21:36:06Z', '2018-02-11T21:39:05Z'
+                    start_time, end_time = '2018-03-11T18:43:08Z', '2018-03-12T18:46:29Z'
```

Comment:
Ugr, no other way? This seems to make re-record difficult.

Reply:
I will open an issue and address it separately once i know more about the filter pattern